pages/contribute/propose-rules.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Suggest new rules
+---
+
+Anyone can suggest new rules on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+Whether you have a single great idea for an accessibility rule or you work for an organization that wants to contribute a whole repository of rules, we welcome the contribution.
+
+When you submit a rule, we expect you to be responsible for the rule the whole way through the review process, including adjusting the rule as you receive feedback from others. It will take some time and effort, but the learning experience in it is great.
+
+If you are not ready to carry a rule all the way through the process, you can partner up with someone else who wants to do it. Partnering up with someone outside of your own organization for rule writing will sharpen the rule, which is why we often do that in our work.
+
+## Submit an idea for discussion
+
+For rule contributors with less GitHub experience, it is often easiest to start the rule design in a [GitHub issue](https://github.com/act-rules/act-rules.github.io/issues). This allows for easy editing and discussions with others, while the rule is taking shape.
```

Comment:
Is there a template for suggesting a new rule?

Reply:
Added to next section.